# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 8c9ba927121c
# Full Hash: 8c9ba927121cd7f1cb5b31ffb86addc4c34b59a8
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-30 21:19:54
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - When line-clamp is in effect, make text-wrap:balance consider only the lines up to the clamp limit. r=emilio
#   
#   This corresponds to how Chrome behaves, and passes the test they included in WPT.
#   
#   It's unclear to me whether this behavior actually follows from the current spec
# ==============================================================================

diff -r 77fce08f9637 -r 8c9ba927121c modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Sat Sep 30 15:53:12 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Sat Sep 30 15:53:13 2023 +0000
@@ -8783,6 +8783,11 @@
   value: 10
   mirror: always
 
+- name: layout.css.text-wrap-balance-after-clamp.enabled
+  type: bool
+  value: true
+  mirror: always
+
 # Support for the css Zoom property.
 - name: layout.css.zoom.enabled
   type: RelaxedAtomicBool